internal/controller/serverset/bootvolume_controller.go

```diff
@@ -118,8 +118,8 @@ func fromServerSetToVolume(cr *v1alpha1.ServerSet, name string, replicaIndex, ve
 			Namespace: cr.Namespace,
 			Labels: map[string]string{
 				serverSetLabel: cr.Name,
-				fmt.Sprintf(indexLabel, resourceBootVolume):   fmt.Sprintf("%d", replicaIndex),
-				fmt.Sprintf(versionLabel, resourceBootVolume): fmt.Sprintf("%d", version),
+				fmt.Sprintf(indexLabel, cr.GetName(), resourceBootVolume):   fmt.Sprintf("%d", replicaIndex),
```

Comment:
Why no use `strconv.Itoa` instead of `fmt.Sprintf`? I saw that you used that in other places.